fe/fe-core/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -1045,12 +1045,23 @@ private static void initColumns(Table tbl, List<ImportColumnDesc> columnExprs,
             return;
         }
 
+        Set<String> tmpSet = Sets.newHashSet();
+        for (ImportColumnDesc importColumnDesc : copiedColumnExprs) {
+            if (importColumnDesc.getExpr() == null) {
+                tmpSet.add(importColumnDesc.getColumnName());
+            }
+        }
+
         // init slot desc add expr map, also transform hadoop functions
         for (ImportColumnDesc importColumnDesc : copiedColumnExprs) {
             // make column name case match with real column name
             String columnName = importColumnDesc.getColumnName();
-            String realColName = tbl.getColumn(columnName) == null ? columnName
-                    : tbl.getColumn(columnName).getName();
+            String realColName;
+            if (tbl.getColumn(columnName) == null || tmpSet.contains(columnName) ){
```

Comment:
It didn't make sense.
Only two case make tbl.getColumn(columnName) != null
1、The columnName match.
2、The columnName match with case insensitive.
And you already change exprMap to case insensitive map, so checking tmpSet.contains(columnName) seems no effect at all.

Reply:
such case, 

CREATE TABLE `record` (
 `id` varchar(50) NOT NULL ,
 `SS` varchar(3) NULL 
) ENGINE=OLAP
UNIQUE KEY(`id`)
DISTRIBUTED BY HASH(`id`) BUCKETS 10;

LOAD LABEL test.record
( 
 DATA INFILE ("hdfs://172.0.0.9/tmp/part0.parq")
 INTO TABLE record format as parquet
 (id, ss) 
 SET
 ( 
	id = id, 
	SS = ss
 )
) WITH BROKER 'Broker_Doris' ( "username" = "hadoop" );

copiedColumnExprs has entrys ("id", "ss", "id = id", "SS = ss"), 
exprMap has entrys ("id = id", "SS = ss"), 
tmpSet has keys ("id", "ss")

1. if don't check tmpSet.contains(columnName), realName will be SS. realName(SS) will be sent to BE. BE uses realName(SS) to match parquet file's SS column, and an error will be occur，parquet file's meta has only ss column
2. if use tmpSet.contains(columnName), realName will be ss

Reply:
How about :
if (tbl.getColumn(columnName) == null || importColumnDesc.getExpr() == null){
    realColName = columnName;
} else {
    realColName = tbl.getColumn(columnName).getName();
}

Reply:
yes, it works and I change it.